inst/rmarkdown/templates/template_sfp_en/skeleton/skeleton.Rmd

```diff
@@ -10,6 +10,9 @@ params:
   version_number: "{{{version_number}}}"
   language: {{{language}}}
   theme: {{{theme}}}
+  dependencies_protocolcode: [sfp_4##_en, sfp_3##_en]
+  dependencies_versionnumber: [YYYY.NN, YYYY.NN]
+  dependencies_params: !r list(list(width = 5, length = 5), list())
```

Comment:
> `dependencies_params: !r list(list(width = 5, length = 5), list())`

I suggest to add some dummy examples here as well; the current format will scare users. Probably preset values such as `params$templatedummy1` and `params$templatedummy2` will not conflict with actual parameters used in the dependencies.